src/WeatherData.js

```diff
@@ -0,0 +1,25 @@
+const weatherData = {
```

Comment:
tohle je celé nepoužité, že? :) používala jsi pro testování/vývoj počítám. To se dělá, ale tak jsi jsi to importovala moc ne :D většinou se to dělá tak že máš API vrstvu a řešíš to na ní:

```
export const getWeather = (city) => fetch (
      API_BASE_FORECAST + API_KEY + '&q=' + city
    )
    .then((response) => response.json())
```
a pak v takovém api.js souboru jen prohodíš/zakomentuješ
```
export const getWeather = (city) => Promise.resolve(weatherData);
```

tím pádem zbytek aplikace neví co se děje v getWeather. Říká se tomu Single responsibility principle https://en.wikipedia.org/wiki/Single-responsibility_principle